### PR TITLE
1622 add sites table

### PR DESCRIPTION
**Pull request by katerina-kossler**
Add Sites table with site_code (string)
Add join table between Sites and Resources for many-to-many relationship

### CODE REVIEW

**Review by jjfreund — CHANGES_REQUESTED** [review]
The migration looks good. I think we also want a many-many relationship between sites and categories, as they will also be site-specific.

We also will want to add the rails domain object for sites (and modify resources & categories) but that can be done in a different commit.

File: db/migrate/20200928034548_create_sites.rb [hunk 1]
```diff
@@ -0,0 +1,7 @@
+class CreateSites < ActiveRecord::Migration[5.2]
+  def change
+    create_table :sites do |t|
+      t.column :site_code, :integer, default: 'sfsg'
```
Comment (jjfreund) [inline db/migrate/20200928034548_create_sites.rb:4]: This should be a string no?

**Review by jjfreund — CHANGES_REQUESTED** [review]

File: db/migrate/20200928034548_create_sites.rb (hunk 1 above)
Comment (katerina-kossler) [reply db/migrate/20200928034548_create_sites.rb]: Yes!

**Review by jjfreund — APPROVED** [review]
lgtm! Not sure if we need the default value for site code but that shouldn't be harmful.

**Pull request merged**